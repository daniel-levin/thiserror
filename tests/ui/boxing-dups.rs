use thiserror::Error;

#[derive(Error, Debug)]
pub enum Dups {
    #[error(transparent)]
    Twice(#[boxing] #[boxing] #[from] std::io::Error),
}

fn main() {}
