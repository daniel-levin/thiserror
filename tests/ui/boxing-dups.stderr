error: duplicate #[boxing] attribute
 --> tests/ui/boxing-dups.rs:6:21
  |
6 |     Twice(#[boxing] #[boxing] #[from] std::io::Error),
  |                     ^^^^^^^^^
